src/components/Footer/Footer.js

```diff
@@ -1,7 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+import FooterDataModal from '../Modal/FooterDataModal';
+import {
+  StyleFooter,
+  FooterContainer,
+  FooterInner,
+  FooterWrapper,
+  Ul,
+  Li,
+  FooterDiv,
+  SlideDiv,
+  MapBtn,
+  MapBtnSpan,
+  MapBtnLink,
+  FooterWrapperDiv,
+  FooterLink,
+} from './StyleFooter';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faGlobe,
+  faChevronUp,
+  faMap,
+  faListUl,
+} from '@fortawesome/free-solid-svg-icons';
 
 const Footer = () => {
-  return <div>Footer</div>;
+  const [isMap, setIsMap] = useState(false);
+  const [isOpenModal, setIsOpenModal] = useState(false);
+
+  const handleModal = () => {
+    setIsOpenModal(!isMap);
+  };
+  return (
+    <StyleFooter>
+      <MapBtn onClick={() => setIsMap(!isMap)}>
+        {isMap === true ? (
+          <MapBtnLink to={`/main/?map_open=${isMap}`}>
+            <MapBtnSpan>목록 보기</MapBtnSpan>
+            <FontAwesomeIcon icon={faListUl} />
+          </MapBtnLink>
+        ) : (
+          <MapBtnLink to={`/main/?map_open=${isMap}`}>
+            <MapBtnSpan>지도 표시하기</MapBtnSpan>
+            <FontAwesomeIcon icon={faMap} />
+          </MapBtnLink>
+        )}
+      </MapBtn>
+      <FooterContainer isMap={isMap}>
+        <FooterInner>
+          <FooterWrapper>
+            <FooterWrapperDiv>
+              © 2023 Space Around, Inc
+              <Ul>
+                {ETC.map(etc => {
+                  return (
+                    <Li key={etc.id}>
+                      <FooterLink to="/">{etc.value}</FooterLink>
+                    </Li>
+                  );
+                })}
+              </Ul>
+            </FooterWrapperDiv>
+            {RightEtc.map(rightEtc =>
+              rightEtc.id !== '3' ? (
+                <SlideDiv key={rightEtc.id}>
+                  <span>{rightEtc.icon}</span>
+                  {rightEtc.value}
+                </SlideDiv>
+              ) : (
+                <SlideDiv key={rightEtc.id}>
+                  {rightEtc.value}
+                  <span
+                    onClick={() => {
+                      handleModal();
+                    }}
```

Comment:
```suggestion
                    onClick={handleModal}
```
인자로 전달할 값이 없다면 위와 같이 구현하셔도 됩니다